config/olm/openshift/0.2.1/manifests/dynatrace-kubernetes-monitoring_rbac.authorization.k8s.io_v1_rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: dynatrace-kubernetes-monitoring
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: dynatrace-kubernetes-monitoring
+subjects:
+- kind: ServiceAccount
+  name: dynatrace-kubernetes-monitoring
+  namespace: dynatrace
```

Comment:
Again, were you able to test it? Can you try in a namespace that is not "dynatrace". From this line, I'd expect it to fail, wouldn't it?